code/game/area/Space_Station_13_areas.dm

```diff
@@ -729,9 +729,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "Prison Wing"
 	icon_state = "sec_prison"
 
-/area/security/prison/toilet
-	name = "Prison Toilet"
-	icon_state = "sec_prison"

```

Comment:
The other maps still use this area, leave it in to fix most of the travis errors.

Reply:
I am planning on editing the other maps as well

Reply:
You arent doing it in this PR, so leave it in because it will fuck up the other maps.

Reply:
I was gonna do it in this PR though, I just am being slow about it